packages/compass-query-bar/src/components/option-editor/option-editor.jsx

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import AceEditor from 'react-ace';
 import 'ace-builds';
-import 'ace-builds/webpack-resolver';
```

Comment:
I spent hours and hours trying to get webpack-resolver to work. It kept seeing ace.config as undefined. In fact the "ace" global it saw had virtually nothing on it. In the end I realised we don't need it and it has potential drawbacks because it basically requires everything possible and then probably relies on webpack to remove the unused parts.

The downside then is that you kinda have to be sure everything the themes/plugins/modes/autocompletion/whatever will want to use is imported. Which is giving me a lot of anxiety.